cobrawap/pipeline/stage04_wave_detection/Snakefile

```diff
@@ -59,7 +59,7 @@ use rule template as optical_flow with:
         script = SCRIPTS / 'optical_flow.py'
     params:
         params('alpha', 'max_Niter', 'convergence_limit', 'gaussian_sigma',
-               'derivative_filter', 'use_phases', config=config)
+               'derivative_filter', 'use_phases', 'n_jobs', 'max_padding_iterations', config=config)
```

Comment:
These parameters would also need to be added to the config_template.yaml file.
I think the number of jobs isn't really a parameter that should be set by the user, but rather set automatically by inferring the available resources, for example, by using `multiprocessing.cpu_count()`

Reply:
Ok we will modify the config_template.yaml file coherently.
It may depend on the specific user and platform. 
In addition, `multiprocessing.cpu_count()` does not differentiate between physical and logical cores. Thus allowing the optical_flow.py script to use all the output of `multiprocessing.cpu_count()` may not result into computational advantage and may interfere with other processes running.
For this reason we suggest the usage of psutil library and a ratio of `psutil.cpu_count(logical = False)` output to set the number of parallel processes based on the number of physical cores, in case the user does not set explicitly the n_jobs parameter (which can be set to None as default).